SRTNet.h

```diff
@@ -75,11 +75,12 @@ class SRTNet {
      * @param overhead % extra of the BW that will be allowed for re-transmission packets
      * @param mtu sets the MTU
      * @param psk Optional Pre Shared Key (AES-128)
+     * @param singleSender set to true to accept just one sender to connect to the server
```

Comment:
```suggestion
     * @param singleSender set to true to accept just one sender to connect to the server, otherwise the server will keep waiting and accepting more incoming sender connections
```